src/clients/QBittorrent.ts

```diff
@@ -224,6 +225,14 @@ export default class QBittorrent implements TorrentClient {
 		autoTMM: boolean;
 		category: string;
 	}> {
+		const { dataDirs } = getRuntimeConfig();
+		if (dataDirs.length > 0) {
```

Comment:
this conditional can be searchee-based to make it more future proof
```suggestion
		if (searchee.path) {
```